<launch>

<node pkg="joy" type="joy_node" name="joy_node">
	<param name="dev" value="/dev/input/js1" type="str" />
</node>
<node pkg="navi_teleop" type="teleop_gamepad" name="teleop_gamepad">
	<param name="axis_linear" value="1"/> <!-- game pad axis-->
	<param name="axis_angular" value="2"/> 
	<param name="scale_angular" value="2"/> 
	<param name="scale_linear" value="1"/> <!-- m/s -->
</node>


</launch>
